daemon/pod.go

```diff
@@ -1004,7 +1004,8 @@ func hyperHandlePodEvent(vmResponse *types.VmResponse, data interface{},
 		if mypod.Status == types.S_POD_RUNNING { // not received finished pod before
 			stopLogger(mypod)
 			mypod.Status = types.S_POD_FAILED
-			mypod.SetContainerStatus(types.S_POD_FAILED)
+			mypod.FinishedAt = time.Now().Format("2006-01-02T15:04:05Z")
+			mypod.SetPodContainerStatus(types.S_POD_FAILED)
 		}
```

Comment:
https://github.com/hyperhq/runv/blob/master/hypervisor/pod.go#L152

within `SetPodContainerStatus`, the `mypod.status` and `mypod.FinishedAt` are both set. This is why I said here is a bug